src/components/Sidebar.jsx

```diff
@@ -0,0 +1,65 @@
+import { useState } from "react";
+
+export default function Sidebar({
+  notes,
+  activeNoteId,
+  createNote,
+  setActiveNote,
+  deleteNote,
+}) {
+  const [hoveredNoteId, setHoveredNoteId] = useState();
+
+  const createNoteTitle = (noteBody) => {
+    return noteBody[0] === "#"
+      ? noteBody.split("\n")[0].slice(1)
+      : noteBody.split("\n")[0];
+  };
+
+  return (
+    <aside>
+      <div className="flex items-center justify-around py-2 bg-white">
+        <h4 className="text-2xl font-bold">Notes</h4>
+        <button
+          className="rounded px-3 py-1 bg-primary-dark text-white hover:bg-[#2f314a]"
+          onClick={createNote}
+        >
+          +
+        </button>
+      </div>
+      <div className="flex flex-col font-medium">
+        {notes.map((note) => {
+          return (
+            <nav
+              onClick={() => {
+                setActiveNote(note);
+              }}
```

Comment:
I think you don't need these curly brackets

```suggestion
              onClick={() =>  setActiveNote(note)}
```